sigs.yaml

```diff
@@ -42,32 +31,29 @@ sigs:
             - https://raw.githubusercontent.com/kubevirt/kubevirt-tutorial/main/OWNERS
     - dir: sig-storage
       name: storage
+      label: sig/storage
       subprojects:
-        - name: containerized-data-importer
+        - name: kubevirt
```

Comment:
if you intend to point out that there's a storage association inside kubevirt (which IMHO is true), then please add pointers to owners files concerning storage portions inside kubevirt/kubevirt in the below owners section.

Example (note that this OWNERS file doesn't exist yet):
* https://github.com/kubevirt/kubevirt/blob/61853c7e26d6a38a666ffa4bdd6f6cf3fe8b4afe/pkg/storage/OWNERS

Reply:
ACK understood

Reply:
BTW, why do you want to remove the cdi subproject here?

Reply:
Reverted, I thought we were using `kubevirt` as a generic subproject when listing `OWNERS` files from multiple different projects.